### DISCUSSION TITLE
How do I get the YOLO badge in github?

### DISCUSSION THREAD

**Discussion by thedrinkingmaster** [community/community#68986] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I tried making a pull request and merge it but i still did not get a the badge, why is that?

**Comment by joywin2003** [comment]
You need to make a fork of that repository you are using and then you try to make a pull request, then you need to add a reviewer and before he accepts that. you need to merge and there you go you'll get the yolo badge

**Comment by supuna97** [comment]
to get the YOLO badge, you need to merge the PR (pull request) directly with the public code repository without verification.

**Comment by RanvilleCarlo** [comment]
To obtain the YOLO badge, you must integrate the PR (pull request) without verification straight into the public code repository.

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
